src/Symfony/Component/Console/Tests/Input/ArgvInputTest.php

```diff
@@ -594,4 +594,260 @@ public static function provideGetRawTokensTrueTests(): iterable
         yield [['app/console', '--no-ansi', 'foo:bar', 'foo:bar'], ['foo:bar']];
         yield [['app/console', '--no-ansi', 'foo:bar', '--', 'argument'], ['--', 'argument']];
     }
+
+    /**
+     * @dataProvider unparseProvider
+     */
+    public function testUnparse(
+        ?InputDefinition $inputDefinition,
+        ArgvInput $input,
+        ?array $parsedOptions,
+        array $expected,
```

Comment:
Please keep the signature on one line. We use multiline signatures only in case we have promoted properties.